hera_cal/frf.py

```diff
@@ -10,16 +10,26 @@
     HAVE_UVTOOLS = False
 
 from . import utils
-
+import scipy.interpolate as interp
```

Comment:
You only ever use `interp1d` from this module, so why not replace this with `from scipy.interpolate import interp1d`?